projects/plugins/jetpack/_inc/client/newsletter/subscriptions-settings.jsx

```diff
@@ -169,6 +179,29 @@ function SubscriptionsSettings( props ) {
 						</FormFieldset>
 					}
 				</SettingsGroup>
+
+				{ isSubscriptionsActive && isSubscriptionSiteFeatureEnabled && (
+					<SettingsGroup
+						hasChild
+						disableInOfflineMode
+						disableInSiteConnectionMode
+						module={ subscriptions }
+					>
+						<p>Enable automatic insertion of the Subscribe block into the theme</p>
```

Comment:
Decided to not wrap it with `__` to not bother translators since it still may change.